hermes/modules/vectorgenerator.py

```diff
@@ -0,0 +1,147 @@
+import data
```

Comment:
Good catch. I will look into it.

From: Alexander Gude <notifications@github.com<mailto:notifications@github.com>>
Reply-To: Lab41/hermes <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, December 17, 2015 at 8:52 AM
To: Lab41/hermes <hermes@noreply.github.com<mailto:hermes@noreply.github.com>>
Cc: J Tiffany <tiffanyj@iqt.org<mailto:tiffanyj@iqt.org>>
Subject: Re: [hermes] Framework (#19)


In hermes/modules/vectorgenerator.py<https://github.com/Lab41/hermes/pull/19#discussion_r47929735>:

> @@ -0,0 +1,147 @@
> +import data


Looks like vectorgenerator import data, and data imports vectorgenerator.

-
Reply to this email directly or view it on GitHub<https://github.com/Lab41/hermes/pull/19/files#r47929735>.



"This e-mail, and any attachments hereto, may contain information that is privileged, proprietary, confidential and/or exempt from disclosure under law and are intended only for the designated addressee(s). If you are not the intended recipient of this message, or a person authorized to receive it on behalf of the intended recipient, you are hereby notified that you must not use, disseminate, copy in any form, or take any action based upon the email or information contained therein. If you have received this email in error, please permanently and immediately delete it and any copies of it, including any attachments, and promptly notify the sender at In-Q-Tel by reply e-mail, fax: 703-248-3001, or phone: 703-248-3000. Thank you for your cooperation."